### ISSUE TITLE
gl_surface_presentation_helper.cc(259)] GetVSyncParametersIfAvailable() failed

### ISSUE THREAD

**Issue by xenoterracide** [vercel/hyper#5857]
- [x] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  Linux manjaro 5.10.56-1-MANJARO #1 SMP PREEMPT Thu Aug 5 10:51:27 UTC 2021 x86_64 GNU/Linux
- **Hyper.app version**:  3.1.2
- **Link of a [Gist](https://gist.github.com/xenoterracide/ca4ec2ed4af125cfe214b799a9f5b5dd/) with the contents of your .hyper.js**: 
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: I don't know how to get this on startup, I'm not sure if it happens otherwise
```

- **The issue is reproducible in vanilla Hyper.app**:  probably? this is the AUR bin distribution, I don't believe they've modified the AUR PKGBUILD in any way, or any meaningful way, it looks like it's just extracting and installing.
https://aur.archlinux.org/packages/hyper-bin

## Issue

I'm not sure if this is even an issue. I just noticed this is in the `-v` output while trying to figure out another issue.



```
[42817:0811/141515.913688:ERROR:gl_surface_presentation_helper.cc(259)] GetVSyncParametersIfAvailable() failed for 2 times!

[42817:0811/141515.917085:ERROR:gl_surface_presentation_helper.cc(259)] GetVSyncParametersIfAvailable() failed for 3 times!
```

probably additionally relevant.

```
wayland 1.19.0
gnome 40
xorg-xwayland 21.1.2
```
xrander output, so the information is available from the system
```
Screen 0: minimum 16 x 16, current 3840 x 2160, maximum 32767 x 32767
XWAYLAND0 connected primary 3840x2160+0+0 (normal left inverted right x axis y axis) 310mm x 170mm
   3840x2160     59.98*+
```

**Comment by LabhanshAgrawal** [comment]
Can't say if this impacts us or not
Looks like a bug from chrome https://bugs.chromium.org/p/chromium/issues/detail?id=879929

**Comment by xenoterracide** [comment]
Yeah, I don't know, I just saw it looking for my problem, I didn't notice anything noticeable. Not sure if it meant acceleration was not working right or not. I leave this one in your capable hands, lol.

**Comment by axelparatre** [comment]
Hello, just a follow up 'cause I'm experiencing the same issue. Thanks for your help

**Comment by the-burrito-triangle** [comment]
Bump: This error still happens with Fedora 41 / Gnome 47 using latest FreeTube (0.21.3 Beta):

$ ./freetube_0.21.3_amd64.AppImage
[1487612:0902/172119.490523:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!

FreeTube seems to work though, so maybe this error isn't a real concern--just adds to startup time when opening.
